src/common/hooks/useDataTablePreferences.ts

```diff
@@ -168,6 +172,10 @@ export function useDataTablePreferences(params: Params) {
       } else {
         setStatus(['active']);
       }
+
+      setTimeout(() => {
+        setArePreferencesApplied(true);
```

Comment:
Do you mind explaining why are we using time-based checks here..? I am always super cautious when it comes to things like this because if timer is too low race condition can mess it up.

Thanks.

Reply:
@beganovich Yes, I understand your point. The only reason I did this is to ensure that this state is updated as the last one, and all re-renderings are done before setting this state to true. But it will work great without this tiny timeout, I just removed it and tested. So, it has been removed!